datasets/us/wy/med_exclusions/crawler.py

```diff
@@ -0,0 +1,75 @@
+from typing import Dict
+
+from zavod import Context, helpers as h
+from rigour.mime.types import PDF
+
+
+from zavod.shed.gpt import run_image_prompt
+
+prompt = """
+ Extract structured data from the following page of a PDF document. Return 
+ a JSON list (`providers`) in which each object represents an medical provider.
+ Each object should have the following fields: `last_name`, `first_name`,
+ `business_name`, `provider_type`, `provider_number`, `city`, `state`, `exclusion_date`.
+ Return an empty string for unset fields.
+"""
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    address = h.make_address(context, city=row.pop("city"), state=row.pop("state"))
```

Comment:
I think if you pass country_code=US as an argument to this, the "full" address will be {city}, {state} which is a bit nicer